apps/web/src/server/lib/async-error-handler.js

```diff
@@ -0,0 +1,15 @@
+// @ts-check
+
+/**
+ * Wrap an asynchronous express middleware such that it catches and handles errors
+ *
```

Comment:
Do me a favour and leave a TODO here and say this should be evaluated again once we migrate to Express 5.